src/Dfe.PlanTech.Web/ServiceCollectionExtensions.cs

```diff
@@ -160,7 +160,7 @@ public static IServiceCollection AddCookies(this IServiceCollection services, IC
     public static IServiceCollection AddCurrentUser(this IServiceCollection services)
     {
         return services
-            .AddSingleton<ICurrentUser, CurrentUser>();
+            .AddScoped<ICurrentUser, CurrentUser>();
```

Comment:
FYI - Scoped is required due to `CurrentUser` now taking the establishment repository via dependency injection

Open to alternatives if available